app/controllers/mentor/practices_controller.rb

```diff
@@ -5,7 +5,9 @@ class Mentor::PracticesController < ApplicationController
   before_action :set_course, only: %i[new]
   before_action :set_practice, only: %i[edit update]
 
-  def index; end
+  def index
+    @practices = Practice.preload(:categories, :products, :reports, :questions).all.order(:id)
```

Comment:
nits
```suggestion
    @practices = Practice.preload(:categories, :products, :reports, :questions).order(:id)
```

どちらでもいいのですが、`all`メソッドは必要なさそうです😊